doc/guide/packages.xml

```diff
@@ -310,6 +310,37 @@ mypackage/test.min.js.gz
       </varlistentry>
     </variablelist>
 
+    <para>The <code>command</code> and <code>environ</code> values can be dynamically
+    taken from a matching open command values. When a value in either <code>command</code>
+    or <code>environ</code> contains a variable wrapped in <code>${}</code>. The variable will be
```

Comment:
@petervo Are these ES6 Template literals, only supporting named variables as the expression? If so, that's worth calling out? https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals